README.md

```diff
@@ -32,7 +32,8 @@ The platform is available at [http://localhost](http://localhost) and the API at
 
 ## Production environment with docker-compose
 
-This project has another docker-compose configuration for the deployment of a production build (file `ui/docker-compose-prod.yml`). This configuration has no `ui` container. Instead the webserver container creates a static production build with the command [`npm run build`](https://create-react-app.dev/docs/available-scripts/).  Which then hosted through nginx. For this reason there is also a another nginx configuration (file `ui/dev/nginx-prod.conf`) with the corresponding 
+This project has another docker-compose configuration for the deployment of a production build (file `ui/docker-compose-prod.yml`). This configuration has no `ui` container. Instead the webserver container creates a static production build with the command [`npm run build`](https://create-react-app.dev/docs/available-scripts/) using a [multi-stage docker file](/usr/share/hunspell) (file `ui/Dockerfile_pb_2`).  Which then hosted through nginx. For this reason there is also a another nginx configuration (file `ui/dev/nginx-prod.conf`) with the corresponding 
```

Comment:
The for the multi-stage Docker file seems to be broken. Please also capitalise "Docker" (it's a name).

"Which then hosted through nginx." is not a full sentence.